### PR TITLE
Contacts form section

### PR DESCRIPTION
**Pull request by Epaltechs**
In this part of my Portfolio Project, I:
- Add Contacts form in the footer
- Put Name field that accepts up to 30 characters
- Put an Email field that accepts only emails.
- Put a Message field that accepts up to 500 characters.
- Make sure all fields have placeholder text.
- Ensure that all fields have the correct type.
- Ensure that a user cannot submit the form if one or more of the fields is empty.
- Arrange that data be collected using the [Formspree service](https://formspree.io/html).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -303,6 +303,32 @@ <h2 class="refer-link">LET'S CONNECT</h2>
                 </div>
             </div>
         </section>
-       </main>
+        <section id="contact-container">
+            <div id="contact-items">
+                <div id="description-contact-box">
+                    <h1 class="contact-title-1">Contact me</h1>
```
Comment (prolajumokeoni) [inline index.html]: Figma Design          |  Your design     
:-------------------------:|:-------------------------:
sssa  | Screenshot_24

- [ ] Kindly change this h1 tag to any hx tag from h2-h6, It is better for your page to have just one h1 tag for semantic purposes
- [ ] Kindly add the appropriate styling before the "contact me" header, there should be a spacing please check the Figma design

File: index.html [hunk 2]
```diff
@@ -303,6 +303,32 @@ <h2 class="refer-link">LET'S CONNECT</h2>
...
+                        placeholder="Emmnanuel Paul" required="required" />
+                    <label class="label" for="email"></label>
+                    <input id="email" class="input-i" name="email" type="email" value="" placeholder="emmapaul247@gmail.com"
+                        required="required" />
+                    <label class="label" id="input-message" for="msg"></label>
+                    <textarea class="input-m" name="msg" id="msg" maxlength="500" cols="20" rows="10"
+                        placeholder="Write your message here" required="required"></textarea>
+                    <button class=" get-touch-btn" type="submit">Get In Touch</button>
+                   </form>
+                   <hr/>
```
Comment (prolajumokeoni) [inline index.html]: - [ ] Kindly delete this horizontal line as it is not required in your design 👍🏿

File: index.html [hunk 3]
```diff
@@ -303,6 +303,32 @@ <h2 class="refer-link">LET'S CONNECT</h2>
                 </div>
             </div>
         </section>
-       </main>
+        <section id="contact-container">
```
Comment (prolajumokeoni) [inline index.html:306]: - [ ] Kindly adjust the background image to look similar to the Figma design, you can adjust the [background size](https://www.w3schools.com/cssref/css3_pr_background-size.asp) of the image 💪🏿

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @Epaltechs  and @RWUBAKWANAYO , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design
✔️  Good PR and Description



## Required Changes ♻️
_Check the comments under the review._
almost-jumpin-gif

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Epaltechs** [comment]
In this commit, I:
- Change the tag in the contact form title from h1 to h5.
- Add margin to the "contact me" header.
- Delete the horizontal line.
- Adjust the background image size and position

**Review by smunozmo — APPROVED** [review]
Hi @Epaltechs, 👋 

Your project is complete!  There is nothing else to say other than... it's time to merge it! 🚀 
Very nice work, congratulations! 🎉 

### APPROVED! 👏 👏 👏 




Cheers and Happy coding! 🤓 

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**